napari/components/dims.py

```diff
@@ -219,25 +182,18 @@ def ndim(self, ndim):
                 self._axis_labels = (
```

Comment:
Have to think about how to do things like this with the evented list - i probably want to modify in place rather than create/destroy

Reply:
yeah, same as above right? you're right that creating a new object will break your connections, so either use slice insertion, or for-loop insertion